osu.Game/Graphics/Containers/Markdown/OsuMarkdownTableCell.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Extensions.Tables;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Overlays;
+
+namespace osu.Game.Graphics.Containers.Markdown
+{
+    public class OsuMarkdownTableCell : MarkdownTableCell
+    {
+        private readonly bool isHeading;
+
+        public OsuMarkdownTableCell(TableCell cell, TableColumnDefinition definition, bool isHeading)
+            : base(cell, definition)
+        {
+            this.isHeading = isHeading;
+            Masking = false;
+            BorderThickness = 0;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider)
+        {
+            var border = new Box
+            {
+                RelativeSizeAxes = Axes.X,
+            };
+
+            // TODO : Change font weight to 700 for heading
```

Comment:
What's the plan for this? Will this be addressed later? Seems like this could be fixed at this point already via a custom `MarkdownTextFlowContainer` - there should be enough extensibility points to do so.

Reply:
I forget about this.

Reply:
Actually, it can't override `CreateTextFlow()` method to use custom `MarkdownTextFlowContainer` because the method is not virtual.

https://github.com/ppy/osu-framework/blob/635e4199dd3e3b4ae8964f057f482aa94862c7ce/osu.Framework/Graphics/Containers/Markdown/MarkdownTableCell.cs#L79-L84

Any suggestions?

Reply:
Open a PR making it virtual then, just like the rest of the components.

Reply:
Yes, I'll do it.